modules/dynamic-operator/plugin/postgresql/main.go

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
Let's move this outside of plugin directory to something like example-plugin

Reply:
ok